src/main/java/io/github/prospector/modmenu/mixin/MixinGameMenuScreen.java

```diff
@@ -17,12 +20,29 @@ public class MixinGameMenuScreen extends Screen {
 	@SuppressWarnings("unchecked")
 	@Inject(at = @At("RETURN"), method = "init")
 	public void drawMenuButton(CallbackInfo info) {
-		this.buttons.add(new ModMenuButtonWidget(100, this.width / 2 - 100, this.height / 4 + 72 - 16, 200, 20,  "Mods (" + ModMenu.getFormattedModCount() + " loaded)"));
+		if (FabricLoader.getInstance().isModLoaded("legacytranslations")) {
+			TranslationStorage var2 = TranslationStorage.getInstance();
+			this.buttons.add(new ButtonWidget(100, this.width / 2 - 100, this.height / 4 + 72 - 16, 98, 20, var2.get("menu.texturepacks")));
+			this.buttons.add(new ModMenuButtonWidget(101, this.width / 2 + 2, this.height / 4 + 72 - 16, 98, 20, var2.get("menu.mods") + " (" + ModMenu.getFormattedModCount() + ")"));
```

Comment:
Where did the "loaded" part go?

Reply:
Added back in, and upgraded LegacyTranslations to be able to handle button text overflow